packages/web-components/src/components/va-official-gov-banner/va-official-gov-banner.scss

```diff
@@ -0,0 +1,304 @@
+}
+
+@media (min-width: 768px) {
+  .header-action {
+    display: none;
+  }
+}
+
+button:not([disabled]):focus {
+  outline: 2px solid var(--color-gold-light);
+}
+
+button[aria-expanded=false], button[aria-expanded=true] {
+  background-image: none;
+}
+
+@media (max-width: 768px) {
+  button {
+    width: 100%;
+  }
+}
+
+button {
+  border: 0;
+  color: #005ea2;
+  font-size: 12px;
+  margin: 0;
+  margin-left: 0.5rem;
+  padding-left: 0;
+  padding-right: 0;
+  position: relative;
+}
+
+@media (max-width: 768px) {
+  button {
+    -webkit-font-smoothing: inherit;
+    background-color: transparent;
+    border: 0;
+    border-radius: 0;
+    box-shadow: none;
+    margin: 0;
+    text-align: left;
+    left: 0;
+    position: absolute;
+    bottom: 0;
+    top: 0;
+    color: #005ea2;
+    display: block;
+    height: auto;
+    line-height: 1.1;
+    padding: 0;
+    text-decoration: none;
+    width: 100%;
+  }
+}
+
+button:hover {
+  color: #1a4480;
+  cursor: pointer;
+}
+
+.button-text {
+  position: absolute;
+  left: -999em;
+  right: auto;
+  text-decoration: underline;
+}
+
+@media (min-width: 768px) {
+  .button-text {
+    position: static;
+    display: inline;
+  }
+}
+
+// Mobile display dropdown/expand more arrow.
+@media (max-width: 768px) {
+  header .header-action::after, header.expanded .header-action::after {
+    background-color: #005ea2;
+  }
+
+  header.expanded .header-action::after, header .header-action::after {
+    background: 0 0;
+    background-color: #005ea2;
+    -webkit-mask: url("../../../../../node_modules/@uswds/uswds/dist/img/usa-icons/expand_more.svg") no-repeat center/contain;
+    mask: url("../../../../../node_modules/@uswds/uswds/dist/img/usa-icons/expand_more.svg") no-repeat center/contain;
+    display: inline-block;
+    height: 18px;
+    width: 18px;
+    content: '';
+    vertical-align: middle;
+    margin-left: 2px;
+  }
+}
+
+// Desktop display dropdown/expand more arrow.
+@media (min-width: 768px) {
+  button::after, button:hover::after {
+    background-color: #005ea2;
+    position: absolute;
+  }
+
+  button::after, header.expanded button::after {
+    background: 0 0;
+    background-color: #005ea2;
+    display: inline-block;
+    height: 18px;
+    width: 18px;
+    content: '';
+    vertical-align: middle;
+    margin-left: 2px;
+    top: -1px;
+  }
+
+  button::after {
+    -webkit-mask: url("../../../../../node_modules/@uswds/uswds/dist/img/usa-icons/expand_more.svg") no-repeat center/contain;
+    mask: url("../../../../../node_modules/@uswds/uswds/dist/img/usa-icons/expand_more.svg") no-repeat center/contain;
+  }
+
+  header.expanded button::after {
+    -webkit-mask: url("../../../../../node_modules/@uswds/uswds/dist/img/usa-icons/expand_less.svg") no-repeat center/contain;
+    mask: url("../../../../../node_modules/@uswds/uswds/dist/img/usa-icons/expand_less.svg") no-repeat center/contain;
+  }
+}
+
+// Mobile display close icon.
+@media (max-width: 768px) {
```

Comment:
The responsive width throughout is based on the [Formation medium screen breakpoint](https://design.va.gov/foundation/breakpoints). The USWDS v3 examples uses `40rem` for a 16px base font so this would be the equivalent to 640px. I simply picked a breakpoint value from Formation that was closest to that value.